Hunting Queries/MultipleDataSources/STRONTIUM_IOC_RetroHunt.yaml

```diff
@@ -0,0 +1,29 @@
+// Name: Retrospective hunt for STRONTIUM IP IOCs 
+// Description: This query searches for activity related to the STRONTIUM threat actor IOCs published in 
+// https://msrc-blog.microsoft.com/2019/08/05/corporate-iot-a-path-to-intrusion/. The query is scoped in 
+// the time window that these IOCs were active.
+//  
+// Id: b8b7574f-1cd6-4308-822a-ab07256106f8
+//
+//
+// Severity: High
+//
+// QueryPeriod: between (startofday(datetime(02/01/2019)) .. endofday(datetime(08/05/2019))
+//
+//
+// DataSource: #CommonSecurityLog, #SecurityAlert  DataTypes: #NetworkDeviceLogs, #ASCIoT
+//
+// Tactics: #CommandAndControl
+//
+let STRONTIUM_IPS = dynamic(["82.118.242.171" , "167.114.153.55" , "94.237.37.28", "31.220.61.251" , "128.199.199.187" ]);
+(union isfuzzy=true
+(CommonSecurityLog
+| where TimeGenerated between (startofday(datetime(02/01/2019)) .. endofday(datetime(08/05/2019)))
+| where SourceIP in (STRONTIUM_IPS) or DestinationIP in (STRONTIUM_IPS)
+),
+(SecurityAlert
+| where TimeGenerated between (startofday(datetime(02/01/2019)) .. endofday(datetime(08/05/2019)))
+| extend RemoteAddress=tostring(parse_json(ExtendedProperties)["RemoteAddress"]
+| where RemoteAddress in (STRONTIUM_IPS) or RemoteAddress in (STRONTIUM_IPS)
```

Comment:
Did you mean to do something other than RemoteAddress?  Otherwise you duplicate the same with the OR.